pykickstart/commands/bootloader.py

```diff
@@ -211,7 +205,8 @@ class F12_Bootloader(F8_Bootloader):
 
     def _getParser(self):
         op = F8_Bootloader._getParser(self)
-        op.add_argument("--lba32", dest="forceLBA", deprecated=True, action="store_true")
+        op.add_argument("--lba32", dest="forceLBA", action="store_true",
+                        deprecated=True, version="Fedora12")
```

Comment:
That is gross, too.  Okay I've looked around at the code a bit and I see we already treat deprecated= as both a bool and a version constant.  So let's:

(1) Change all the calls of deprecated=True to deprecated=<version constant>.  We'll still have to accept deprecated=True/False in options.py, and we probably can't do anything clever to figure out its help output, but I've supported it for so long it's gotta stay.
(2) Add a version=<version constant> argument to remove_argument.
(3) Somewhere add a conversion function that takes a version constant as an input and spits out a string suitable for feeding to help, and calling that in add_argument and remove_argument.  That way the help stuff doesn't leak out into all other functions.

Does this take care of everything?